probe-dictionary/explore.js

```diff
@@ -16,6 +16,55 @@ var gDatasetMappings = null;
 var gView = null;
 var gDetailViewId = null;
 
+$(document)
+  .ready(function () {  
+    // Permalink control
+    $(".permalink-control")
+      .append(
+        '<div class="input-group">' +
+        '    <span class="input-group-btn"><button style="background:none; cursor:pointer; padding: 7.5px 8px; font-size: 0.9375rem; font-weight: normal; z-index:auto; line-height: 1.5; position:static; border:none; color:rgba(255,255,255,.75); font-family: "Segoe UI", "Source Sans Pro", Calibri, Candara, Arial, sans-serif;" type="button" class="btn btn-default" title="Get Permalink"><i class="fa fa-link"></i> Get Permalink</button></span>' +
+        '    <input type="text" class="form-control">' +
+        '</div>'
+      );
+    $(".permalink-control input")
```

Comment:
Do we need this workaround in this case? Does selection and copying work without it?